src/pages/init.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react'
+import { GetServerSideProps, NextPage } from 'next'
+import { gql, request } from 'graphql-request'
+import { Response } from 'express'
+import { NEXT_PUBLIC_GQL_BACKEND_URL } from '../server/environment'
+import { InitPageQuery } from '../generated/graphql'
+import { INITIALIZED_COOKIE_NAME } from '../lib/withRedirectIfNotInitialized'
+import AlreadyInitialized from '../components/initialization/AlreadyInitialized'
+import InitializationSuccessful from '../components/initialization/InitializationSuccessful'
+import InitializationInProgress, {
+  SubmissionStatus,
+} from '../components/initialization/InitializationInProgress'
+
+interface InitPageProps {
+  isInitialized: boolean
+}
+
+export const getServerSideProps: GetServerSideProps<InitPageProps> = async (
+  context
+) => {
+  const { req, res } = context
+
+  if (req.cookies[INITIALIZED_COOKIE_NAME]) {
+    return { props: { isInitialized: true } }
+  }
+
+  const { isInitialized } = await request<InitPageQuery>(
+    NEXT_PUBLIC_GQL_BACKEND_URL,
```

Comment:
Is `NEXT_PUBLIC_GQL_BACKEND_URL` only used from within `GetServerSideProps`? if so, should we change it back to `GQL_BACKEND_URL`?

I don't remember exactly what was it, but in the past, I've had issues with `NEXT_PUBLIC_*`. Maybe things have changed?

What's the benefit of prefixing with `NEXT_PUBLIC`, if it's not used on the browser side?